res/openvpn-gui-res-zh-hant.rc

```diff
@@ -200,188 +232,225 @@ END
     IDS_MENU_SERVICEONLY_RESTART "重新連線"
-    IDS_MENU_ASK_STOP_SERVICE "您想要斷線嗎 (停止  OpenVPN 服務)?"
+    IDS_MENU_ASK_STOP_SERVICE "您想要斷線（停止 OpenVPN 服務）嗎？"
 
     /* Logviewer - Resources */
-    IDS_ERR_START_LOG_VIEWER "啟動 log-viewer 失敗: %s"
-    IDS_ERR_START_CONF_EDITOR "啟動 config-editor 失敗: %s"
+    IDS_ERR_START_LOG_VIEWER "記錄檔檢視器啟動失敗: %s"
+    IDS_ERR_START_CONF_EDITOR "設定檔編輯器啟動失敗: %s"
 
     /* OpenVPN */
-    IDS_ERR_MANY_CONFIGS "OpenVPN GUI 並無提供超過 %d 的configs. 如果您有需要,請聯繫作者."
-    IDS_ERR_ONE_CONN_OLD_VER "若您使用OpenVPN 2.0-beta6以前的舊版本,您只能一次連線一個."
-    IDS_ERR_STOP_SERV_OLD_VER "當OpenVPN 服務正在使用時,您無法使用OpenVPN GUI啟動連線 (with OpenVPN 1.5/1.6). 若您需要使用OpenVPN GUI,請停止OpenVPN服務."
-    IDS_ERR_CREATE_EVENT "CreateEvent failed on exit event: %s"
-    IDS_ERR_UNKNOWN_PRIORITY "未知優先名稱: %s"
-    IDS_ERR_LOG_APPEND_BOOL "Log檔案追加標誌(例如 '%s')必須為 '0' or '1'"
-    IDS_ERR_GET_MSIE_PROXY "無法取得MSIE proxy 設定."
-    IDS_ERR_INIT_SEC_DESC "初始化安全描述失敗."
-    IDS_ERR_SET_SEC_DESC_ACL "設定安全描述失敗."
-    IDS_ERR_CREATE_PIPE_OUTPUT "建立hOutputWrite管道失敗."
-    IDS_ERR_CREATE_PIPE_INPUT "建立hInputRead管道失敗."
-    IDS_ERR_DUP_HANDLE_OUT_READ "hOutputRead重複處理失敗."
-    IDS_ERR_DUP_HANDLE_IN_WRITE "hInputWrite重複處理失敗."
-    IDS_ERR_CREATE_PROCESS "建立程序失敗, exe='%s' cmdline='%s' dir='%s'"
-    IDS_ERR_CREATE_THREAD_STATUS "建立執行序顯示狀態視窗失敗."
-    IDS_NFO_STATE_WAIT_TERM "目前狀態: 等待中止OpenVPN"
-    IDS_NFO_STATE_CONNECTED "目前狀態: 已連接"
-    IDS_NFO_NOW_CONNECTED "%s 目前已連接."
-    IDS_NFO_ASSIGN_IP "指定IP: %s"
-    IDS_ERR_CERT_EXPIRED "無法連接,因為您的認證已過期或是系統時間不正確."
-    IDS_ERR_CERT_NOT_YET_VALID "無法連接,因為您的認證尚未生效.請檢查您的系統時間是正確的."
+    IDS_ERR_MANY_CONFIGS "OpenVPN GUI 不支援超過 %d 組連線設定檔。若您有需要，請聯繫作者。"
+    IDS_NFO_NO_CONFIGS "沒有找到可讀取的連線設定檔。\n"\
+                       "請使用選單的「匯入設定檔…」，或將設定檔複製到「%s」或「%s」。"
+    IDS_ERR_CONFIG_NOT_AUTHORIZED "啟動此連線 (%s) 必須要有「%s」群組的成員資格。"\
+                                  "請連絡您的系統管理員。\n"
+    IDS_ERR_CONFIG_TRY_AUTHORIZE  "啟動此連線 (%s) 必須要有「%s」群組的成員資格。"\
+                                  "\n\n"\
+                                  "您想要把自己加入到此群組嗎？\n"\
+                                  "可能需要輸入管理密碼或管理員同意才能進行此行為。"
+    IDS_NFO_CONFIG_AUTH_PENDING   "啟動此連線 (%s) 必須要有「%s」群組的成員資格。"\
+                                  "\n\n"\
+                                  "請完成先前的授權對話窗。"
+    IDS_ERR_ADD_USER_TO_ADMIN_GROUP "將使用者加入至「%s」群組失敗。"
+    IDS_ERR_ONE_CONN_OLD_VER "若您使用 OpenVPN 2.0-beta6 以前的舊版本，同時僅能有一個連線。"
+    IDS_ERR_STOP_SERV_OLD_VER "OpenVPN 1.5/1.6 的服務執行中時，將無法使用 OpenVPN GUI 進行連線。若您想使用 OpenVPN GUI，請先停止 OpenVPN 服務。"
+    IDS_ERR_CREATE_EVENT "發生結束事件時 CreateEvent 失敗: %s"
+    IDS_ERR_UNKNOWN_PRIORITY "未知的優先權名稱: %s"
+    IDS_ERR_LOG_APPEND_BOOL "附加記錄檔的旗標（例如「%s」）必須為「0」或「1」"
+    IDS_ERR_GET_MSIE_PROXY "無法取得 MSIE 的 Proxy 設定。"
+    IDS_ERR_INIT_SEC_DESC "InitializeSecurityDescriptor 失敗。"
+    IDS_ERR_SET_SEC_DESC_ACL "SetSecurityDescriptorDacl 失敗。"
+    IDS_ERR_CREATE_PIPE_OUTPUT "建立 hOutputWrite 管道失敗。"
+    IDS_ERR_CREATE_PIPE_INPUT "建立 hInputRead 管道失敗。"
```

Comment:
This would be another issue, but I don't think we should expose function names to error messages.
Leave it in English if we want to make it searchable regardless l10n; or translate it to natural language to make it understandable.